### ISSUE TITLE
Redundant HTTP upload when saving child-object with attachment

### ISSUE THREAD

**Issue by lrstry** [odoo/odoo#68174]
Impacted versions: 

- at least > 14.0 

Steps to reproduce:

- Prerequisite: Model with Parent/Child relationship (One2many), Child has field.Binary with attachment=True
- Upload attachment in Child-Model, Save Child (HTTP Upload is being processed)
- Save Parent (Same HTTP Upload is being processed again)

Current behavior:

- Same HTTP Upload is being processed twice in total

Expected behavior:

- HTTP Upload is being processed once
- This is crucial for long-running uploads due to big file sizes or slow connections

Video/Screenshot link (optional):

Support ticket number submitted via odoo.com/help (optional):

**Comment by pivi-odoo** [comment]
Thank you for reporting this performance issue.

As part of our effort to clean up and triage performance-related issues, and because this issue concerns an older version of Odoo, we are closing it.

If you still experience the same problem on a currently supported version, please open a new issue with updated reproduction steps, relevant logs, and performance measurements.
Or open a ticket on odoo.com/help if you have an ongoing subscription for faster assistance.

Thank you for your understanding.
